app/views/users/new.html.erb

```diff
@@ -1,21 +1,21 @@
 <h2>New User Registration</h2>
 
-<%= form_tag "/users" do %>
+<%= form_tag "/register" do %>
```

Comment:
We might want to think about using form_with here vs. form_tag. That way we don't have to include all those conditionals to pre-populate the form.

Reply:
Cool, I haven't seen form_with yet